frontend/src/views/app-store/apps/index.vue

```diff
@@ -350,7 +356,7 @@ onMounted(async () => {
     }
     search(req);
     if (isProductPro.value) {
-        const res = await GetCustomAppStoreConfig();
+        const res = await getCurrentNodeCustomAppConfig();
         if (res && res.data) {
             syncCustomAppstore.value = res.data.status === 'enable';
         }
```

Comment:
There don't seem to be any syntax errors or unusual coding practices that would indicate an issue with the provided code snippet. However, it should be noted that this is not a typical Vue.js application; the code snippets you've shared do not appear to fit within a normal Vue project context.

For optimization considerations:
1. Ensure all required imports are included at the top of the file, rather than scattered throughout multiple places.
2. If there's more than one function doing the same task, consider using `async`/`await` for promises.
3. Use local storage instead of cookies for sensitive data like app tags, since cookies can be easily removed or manipulated upon logout in modern browsers and servers.

As stated earlier, I believe these code samples aren’t relevant to a regular Vue project and could potentially cause confusion without specific context about their purpose and use cases.